Automattic/studio
# Studio: Add AI guidelines link

Fixes https://github.com/Automattic/dotcom-forge/issues/7847
Related  https://github.com/Automattic/dotcom-forge/issues/7446

## Proposed Changes

This PR adds AI guidelines URL to studio

## Testing Instructions
1. Navigate to AI assistant tab
2. Ensure that you see a guidelines section like so:
![2024-06-20T13:39:35,499593815+03:00](https://github.com/Automattic/studio/assets/497103/d7573477-cbc0-432e-9056-7fe9328841ff)

4. Ensure that clicking it, navigates you to https://automattic.com/ai-guidelines/



## Pre-merge Checklist
